### PR TITLE
Create bug_report.yml

### PR DESCRIPTION
**Pull request by sdreyer**
# Description

A Github issues template to help formalize required information from contributors about issues they may be having.

Preview Here: https://github.com/featureform/featureform/blob/other/issues-template/sterling/.github/ISSUE_TEMPLATE/bug_report.yml

### Does this correspond to an open issue?

No

### Select type(s) of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [X] Documentation update

# Checklist:

- [X] I have performed a self-review of my code
- [X] I have commented my code, particularly in hard-to-understand areas
- [X] I have made corresponding changes to the documentation
- [X] My changes generate no new warnings
- [N/A] I have added tests that prove my fix is effective or that my feature works
- [X] New and existing unit tests pass locally with my changes
- [X] I have fixed any merge conflicts

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.yml [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+labels: ["bug", "triage"]
+body:
+  - type: textarea
+    id: expected-behavior
+    attributes:
+      label: Expected Behavior
+      description: Please describe the behavior you expected
+    validations:
+      required: true
+  - type: textarea
```
Comment (ahmadnazeri) [inline .github/ISSUE_TEMPLATE/bug_report.yml:13]: can you include images in textareas as well?

File: .github/ISSUE_TEMPLATE/bug_report.yml [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+      label: What mode are you running Featureform in?
+      options:
+        - Local
+        - Hosted
+    validations:
+      required: true
+  - type: dropdown
+    id: python-verison
+    attributes:
+      label: What version of Python are you running?
```
Comment (ahmadnazeri) [inline .github/ISSUE_TEMPLATE/bug_report.yml:39]: Should we add a description that states we only support these versions?

**Review by ahmadnazeri — APPROVED** [review]
LGTM

File: .github/ISSUE_TEMPLATE/bug_report.yml (hunk 2 above)
Comment (sdreyer) [reply .github/ISSUE_TEMPLATE/bug_report.yml:39]: Do we need it? The drop down forces you to choose one of the version and you can only install it on those versions anyways.

File: .github/ISSUE_TEMPLATE/bug_report.yml (hunk 1 above)
Comment (sdreyer) [reply .github/ISSUE_TEMPLATE/bug_report.yml:13]: Yeah it does, just added a tip that you can drag files into it

**Pull request merged**